commands/comment.go

```diff
@@ -109,9 +111,22 @@ func buildCommentFromFlags(repo repository.Repo, commentedUponCommit string) (*c
 	if err != nil {
 		return nil, err
 	}
+
+	date, err := GetDate(*commentDate)
+	if err != nil {
+		return err
```

Comment:
This doesn't build as-is.

It needs to return two values instead of one, so `return nil, err`